homeassistant/components/plaato/binary_sensor.py

```diff
@@ -0,0 +1,60 @@
+"""Support for Plaato Airlock sensors."""
+
+import logging
+
+from pyplaato.plaato import PlaatoKeg
+
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_OPENING,
+    DEVICE_CLASS_PROBLEM,
+    BinarySensorEntity,
+)
+
+from .const import CONF_USE_WEBHOOK, COORDINATOR, DOMAIN
+from .entity import PlaatoEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the Plaato sensor."""
+
+
+async def async_setup_entry(hass, config_entry, async_add_devices):
+    """Set up Plaato from a config entry."""
+
+    if not config_entry.data.get(CONF_USE_WEBHOOK, False):
+        coordinator = hass.data[DOMAIN][config_entry.entry_id][COORDINATOR]
+        if coordinator.data is not None:
+            async_add_devices(
```

Comment:
Please use a guard and return early instead of indenting.

if config_entry.data.get(CONF_USE_WEBHOOK, False):
    return

and then outdent